centrifuge-app/src/utils/formatting.ts

```diff
@@ -80,3 +81,66 @@ export function truncateText(txt: string, len: number) {
   }
   return txt
 }
+
+// FORMAT FUNCTIONS FOR NEW SDK
+
+export function formatDecimal(decimalVal: any, displayDecimals: number, currency?: string): string {
```

Comment:
```suggestion
export function formatDecimal(  amount: CurrencyBalance | TokenBalance | Price | Rate | Decimal | number,
, displayDecimals = 2, currency?: string): string {
```
Would love to see this function types like the other formatting options we had. There will be some type conflicts with Rate/Perquintill but I think you can circumvent the problem by writing the functions in a new file e.g `formatting-sdk.ts`. Then we can likely also keep the names `formatBalance`, `formatBalanceAbbreviated` and `formatPercentage` (I don't think `formatDecimal` really conveys what the function does)